libs/@local/hql/frontend-jexpr/README.md

```diff
@@ -0,0 +1,94 @@
+# Frontend
+
+Collection of HQL frontends, which are languages, that are compiled into HQL-CST queries.
+These frontends are used to provide a more user-friendly way to write queries, than writing HQL-CST queries directly.
+
+In the future, frontends may implement traits from a `hql-frontend-core`, more research needs to be done on the subject, as it is unclear, if using the same I/O for all frontends is even possible.
+
+Currently supported frontends:
+
+- J-Expr: JSON Expression Language - a simple S-Expr like language, based on JSON syntax
+
+Due to a limitation of cargo, we're unable to nest libraries, see: <https://github.com/rust-lang/cargo/issues/6745>, therefore this documentation is in `frontend-jexpr` not in `frontend`.
+
+# J-Expr - JSON Expression Language
+
+J-Expr is a simple S-Expr like language, based on JSON syntax.
+
+## Syntax
+
+### Function Call
+
+A function call is represented as a non-zero JSON array, where the first element is the function to be invoked, and the rest of the elements are the arguments to the function.
+
+```
```

Comment:
```suggestion
```ts
```

Reply:
done in [a83f408](https://github.com/hashintel/hash/pull/4828/commits/a83f4082ecdb45d8cce8ecfbd40f65b28206ae4d)